docs/contributor-guide/how-to-write-docs.md

```diff
@@ -0,0 +1,32 @@
+# How to write docs
+
+When writing user-facing documentation, pick one of the four options:
+1. Tutorial
+2. How-To guide
+3. Explanation
+4. Reference
+
+For more, see descriptions below or [Divio's documentation system manual](https://documentation.divio.com/).
+
+### Tutorial
+
+Tutorials are learning-oriented and ...
+- must walk-through a self-contained domain like "Deduplicating buildings in Cambridge".
+- must provide any necessary configuration and data for the tutorial.
+
+### How-To guide
+
+How-To guides are problem-oriented and ...
+- must answer a specific, domain-agnostic question like "How to stream datasets out of Tamr"
+
+### Explanation
+
+Explanations are understanding-oriented and ...
+- must explain core concepts of Tamr or the Tamr client
+- may include code examples
+
+### Reference
+
+Reference is information-oriented.
```

Comment:
Reference is information-oriented. It is something that users cannot remember and want to be able to refer to, often. Reference provides details, such as configuration parameters for a particular method or call. It never contains tasks, or concepts. Reference is often automatically-generated from code, to ensure it is up-to-date and accurate at all times.